### ISSUE TITLE
Delete tracks from Album page

### ISSUE THREAD

**Issue by seankwilliams** [codefordenver/Comrad#476]
Depends on #281 and #283

Set up an ellipsis similar to the one on the Music Library page. It should be in the right side of each row of the track table and clicking it should open a dropdown with "Edit" and "Delete" options. Clicking "Edit" won't do anything yet, but clicking "Delete" should open the same Delete dialog as the Music Library page.
